package.json

```diff
@@ -19,17 +19,17 @@
     "lint": "eslint --cache .",
     "typecheck": "tsc -p scripts --noEmit && pnpm -r --parallel run typecheck",
     "test": "run-s test-unit test-serve test-build",
-    "test-serve": "vitest run -c vitest.config.e2e.ts",
-    "test-build": "VITE_TEST_BUILD=1 vitest run -c vitest.config.e2e.ts",
+    "test-serve": "nx exec -- vitest run -c vitest.config.e2e.ts",
```

Comment:
`nx exec` is used to wrap any package scripts that that should run through Nx. This way non of the local or CI scripts need to be updated to integrate with Nx.

For example, users continue to use `pnpm run docs` not `nx run docs`.

Reply:
does the nx cli prefix/modify log output of the original command?
does it allow passing additional args like focussing a test?

Reply:
the `nx exec` command executes whatever command you pass to it (including any flags).  The output is rendered with the exact same colors and formatting.  The only modification is that Nx adds a message at the end saying "replayed from cache" or something similar.